tests/schemas/valid/test_section_summary_completion.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
We don't usually create test schemas for new content that is validated by JSON schema. The test schemas test the validation done in code.